include/solvers/mpm_base.tcc

```diff
@@ -554,23 +588,29 @@ void mpm::MPMBase<Tdim>::write_vtk(mpm::Index step, mpm::Index max_steps) {
   }
 
   // VTK state variables
-  for (const auto& attribute : vtk_statevars_) {
-    // Write state variables
-    auto file =
-        io_->output_file(attribute, extension, uuid_, step, max_steps).string();
-    vtk_writer->write_scalar_point_data(
-        file, mesh_->particles_statevars_data(attribute), attribute);
-    // Write a parallel MPI VTK container file
+  for (auto const& vtk_statevar : vtk_statevars_) {
+    unsigned phase_id = vtk_statevar.first;
+    for (const auto& attribute : vtk_statevar.second) {
+      std::string phase_attribute = "P" + std::to_string(phase_id) + attribute;
```

Comment:
Maybe just use `"phase" + phase_id` instead of `"P" + phase_id`. I feel there are many `"P"s` within the code (e.g. particles, pressure, phase) so why not be specific? 
```suggestion
      std::string phase_attribute = "phase" + std::to_string(phase_id) + attribute;
```